tests/hardhat/Prime/PrimeLiquidityProvider.ts

```diff
@@ -304,19 +311,19 @@ describe("PrimeLiquidityProvider: tests", () => {
       expect(Number(balanceB)).to.equal(accruedTokenB);
 
       previousAccruedTokenA = await primeLiquidityProvider.tokenAmountAccrued(tokenA.address);
-      lastAccruedBlockTokenA = await primeLiquidityProvider.lastAccruedBlock(tokenA.address);
+      lastAccruedBlockTokenA = await primeLiquidityProvider.lastAccruedBlockOrSecond(tokenA.address);
       await primeLiquidityProvider.accrueTokens(tokenA.address);
-      currentBlockTokenA = await primeLiquidityProvider.getBlockNumber();
+      currentBlockTokenA = await primeLiquidityProvider.getBlockNumberOrTimestamp();
```

Comment:
Should we also change variables names for block or second in testing too?

Reply:
Changed: https://github.com/VenusProtocol/venus-protocol/pull/407/commits/8e135084bc805d4f88d257a04a4519d13e28e62f